core/src/main/java/io/questdb/griffin/engine/functions/math/GreatestNumericFunctionFactory.java

```diff
@@ -182,14 +192,17 @@ public ObjList<Function> getArgs() {
         @Override
         public long getLong(Record rec) {
             long value = Long.MIN_VALUE;
+            boolean allAreNull = true;
```

Comment:
There is no need for this variable as the `value` is set to `Long.MIN_VALUE` initially.